### PR TITLE
[[ Installer ]] Add html_viewer media folder to installer manifest

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by peter-b** [comment]
@livecode-vulcan review ok d415bfd

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok d415bfd2af8e1662387e47cfc0dbaae9321508ed

**Comment by livecode-vulcan** [comment]
😎 test success d415bfd2af8e1662387e47cfc0dbaae9321508ed
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2700)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2710)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2695)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2711)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2717)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2723)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2735)

File: Installer/package.txt [hunk 1]
```diff
@@ -367,6 +367,9 @@ component Documentation
 		rfolder "ide:Documentation/html_viewer/fonts"
 		rfolder "ide:Documentation/html_viewer/js"
 
+	into [[ToolsFolder]]/Documentation/html_viewer/resources/ place
```
Comment (livecodepanos) [inline Installer/package.txt:370]: I think this should be `into [[ToolsFolder]]/Documentation/html_viewer/resources place` (i.e. without the `/`)

**Pull request merged**